hid-uclogic-rdescs.h

```diff
@@ -0,0 +1,48 @@
+#ifndef UCLOGIC_REPORT_DESCRIPTORS
+#define UCLOGIC_REPORT_DESCRIPTORS
+
+#include <linux/device.h>
+
+/* Size of the original descriptor of WPXXXXU tablets */
+#define UCLOGIC_WPXXXXU_RDESC_ORIG_SIZE	212
+
+/* Size of the original descriptor of the new WP5540U tablet */
+#define UCLOGIC_WP5540U_V2_RDESC_ORIG_SIZE	232
+
+/* Size of the original descriptor of WP1062 tablet */
+#define UCLOGIC_WP1062_RDESC_ORIG_SIZE	254
+
+/* Size of the original descriptor of PF1209 tablet */
+#define UC_LOGIC_PF1209_RDESC_ORIG_SIZE	234
+
+/* Size of the original descriptors of TWHL850 tablet */
+#define UCLOGIC_TWHL850_RDESC_ORIG_SIZE0	182
+#define UCLOGIC_TWHL850_RDESC_ORIG_SIZE1	161
+#define UCLOGIC_TWHL850_RDESC_ORIG_SIZE2	92
+
+/* Size of the original descriptors of TWHA60 tablet */
+#define UCLOGIC_TWHA60_RDESC_ORIG_SIZE0 254
+#define UCLOGIC_TWHA60_RDESC_ORIG_SIZE1 139
+
+/* Report descriptor template placeholder head */
+#define UCLOGIC_PH_HEAD	0xFE, 0xED, 0x1D
+
+/* Report descriptor template placeholder */
+#define UCLOGIC_PH(_ID) UCLOGIC_PH_HEAD, UCLOGIC_PH_ID_##_ID
+#define UCLOGIC_PEN_REPORT_ID	0x07
+
+extern const __u8 uclogic_wp4030u_rdesc_fixed[];
+extern const __u8 uclogic_wp5540u_rdesc_fixed[];
+extern const __u8 uclogic_wp8060u_rdesc_fixed[];
+extern const __u8 uclogic_wp1062_rdesc_fixed[];
+extern const __u8 uclogic_pf1209_rdesc_fixed[];
+extern const __u8 uclogic_twhl850_rdesc_fixed0[];
+extern const __u8 uclogic_twhl850_rdesc_fixed1[];
+extern const __u8 uclogic_twhl850_rdesc_fixed2[];
+extern const __u8 uclogic_twha60_rdesc_fixed0[];
+extern const __u8 uclogic_twha60_rdesc_fixed1[];
+extern const __u8 uclogic_tablet_rdesc_template[];
+extern const __u8 uclogic_ugee_ex07_rdesc_template[];
+extern const __u8 uclogic_buttonpad_rdesc[];
```

Comment:
Could you please move the comments describing these here, from the .c file? And then also group them together with the macros, by the tablet model?

Reply:
I mean group not just the comments, but the variable declarations together with comments.